source/docs/articles/drupal/8.md

```diff
@@ -0,0 +1,32 @@
+---
+use: [8]
+layout: article
+title: Drupal 8 on Pantheon
+---
+
+[Drupal 8 RC1](https://www.drupal.org/drupal-8.0) is now available on Pantheon! You can create Drupal 8 sites on Pantheon, but things may break and our support is limited.
+
+At this time the following platform functionality are not available:
+
+ - Solr
+ - Redis
+ - Launch Check
```

Comment:
Can we make explicit that Launch Check is what shows up on the Status tab?
Maybe change "Launch Check" to "Launch Check on Status tab"